src/lib/pages/account-details/index.tsx

```diff
@@ -79,9 +140,32 @@ const AccountDetailsBody = ({ accountAddress }: AccountDetailsBodyProps) => {
             <Text>Transactions</Text>
             {/* TODO: replace with the truncated Codes table */}
             <Text>Stored Codes</Text>
-            {/* TODO: replace with the truncated Contracts table */}
-            <Text>Contract Instances</Text>
-            {/* TODO: replace with the truncated Admins table */}
+            <Box>
+              <InstantiatedContractTable
+                walletAddress={accountAddress}
+                scrollComponentId={tableHeaderId}
+                totalData={tableCounts.contractsCount}
+                refetchCount={refetchContractsCount}
+                isPreview
+              />
+              {!!tableCounts.contractsCount &&
+                tableCounts.contractsCount >= 5 && (
+                  <ViewMore onClick={() => setTabIndex(TabIndex.Contracts)} />
+                )}
```

Comment:
I think we've already sent `isPreview` and `tableCounts.contractsCount` as props on `Table`. Why don't we move `ViewMore` logic into that component?